ynr/settings/base.py

```diff
@@ -138,6 +138,7 @@ def root(*path):
 )
 
 CELERY_RESULT_BACKEND = "django-db"
+CELERY_TASK_ALWAYS_EAGER = True
```

Comment:
I think I'd add a comment to this line with some very short explanation explaining it's overridden to `False` in production, since it is quite surprising to have this in the base settings (to me, anyway) and not everyone's great at looking through git history.